components/card/src/vwc-card.ts

```diff
@@ -7,14 +7,17 @@ import { property } from 'lit-element/lib/decorators.js';
 import { classMap } from 'lit-html/directives/class-map.js';
 import '@vonage/vwc-button';
 import '@vonage/vwc-icon';
-
+import '@vonage/vwc-elevation';
 
 declare global {
 	interface HTMLElementTagNameMap {
 		'vwc-card': VWCCard;
 	}
 }
 
+const elevationSets = ['0' , '2', '4', '8', '12' , '16' , '24'];
```

Comment:
can be imported from elevation

Reply:
there are non in the elevation comp... 😨

Reply:
we should open a ticket to support ts types of elevation tokens